engine/model_get.go

```diff
@@ -1,3 +1,6 @@
+//revive:disable
+//nolint:wrapcheck // wrapping should be done at the higher level
+//revive:enable
```

Comment:
Is this correct ?

Reply:
Unfortunately yes, 
I wan't to disable wrapcheck on whole this file,
to do that I need to put `//nolint:wrapcheck` on the package
BUT when I do that, then revive is complaining that comment on the package should start with uppercase (in other words it's forcing us to make a documentation comment on a package. 
And the solution for that is to disable and enable revive just for this fragment.